gcovr/configuration.py

```diff
@@ -655,6 +655,21 @@ class UseSortUncoveredPercentAction(GcovrDeprecatedConfigOptionAction):
             "Default is '{default!s}'."
         ),
     ),
+    GcovrConfigOption(
+        "merge_mode_conditions",
+        ["--merge-mode-conditions"],
+        metavar="COND_MERGE_MODE",
```

Comment:
This value is only used in the help, we do not refer to it so we can use the same meta variable for all merge options.
```suggestion
        metavar="MERGE_MODE",
```